R/selections.R

```diff
@@ -172,7 +172,7 @@ NULL
 #' recipes_eval_select(quos, scat, info)
 recipes_eval_select <- function(quos, data, info, ..., allow_rename = FALSE,
                                 check_case_weights = TRUE, call = caller_env()) {
-  ellipsis::check_dots_empty()
+  check_dots_empty()
```

Comment:
```suggestion
  check_dots_empty(call = call)
```

@EmilHvitfeldt actually, may be worth adding the call argument since `check_dots_empty()` accepts it and `recipes_eval_select()` already has it?

Reply:
normally I would agree, but in this case, `check_dots_empty()` should only be triggered if the developer uses `recipes_eval_select()` incorrectly, so I think it is fine that `check_dots_empty()` is the one highlighted